recipe/meta.yaml

```diff
@@ -10,25 +10,28 @@ source:
   md5: fff8a7d7f580b04bacc2ffe7ddf41133
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - ssl_match_hostname  # [py<35]
+    - ssl_match_hostname   # [py<35]
```

Comment:
They all seem to have passed.

I know these proceeded your proposal, but if you don't mind grabbing them too that would be nice.